code/process_sequences.sh

```diff
@@ -23,7 +23,7 @@ then
 exit 1
 fi
 # echo statement to describe contents:
-echo "This is some information about the sequences from the Sars-Cov-2 database"
+echo "This is some information about the sequences for SARS-Cov-2 from BLAST."
```

Comment:
Not from BLAST; BLAST is a software tool for searching for similar sequences. I'd say "for SARS-CoV-2 from NCBI."

Reply:
Oh whoops ha ha ok I'll fix that.